### PR TITLE
feat(ui): change the breadcrumbs on settings modal

### PR DESCRIPTION
**Pull request by baymax-codecare**
**What this PR does** 📖
On settings modal, the breadcrumbs navigation is weird, This PR fixes these weird breadcrumbs.
**Which issue(s) this PR fixes** 🔨
AP-728

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: components/views/settings/breadcrumbs/Breadcrumbs.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<template src="./Breadcrumbs.html"></template>
+<script lang="ts">
+import Vue from 'vue'
+import { mapState } from 'vuex'
+
+export default Vue.extend({
+  computed: {
+    ...mapState(['ui']),
+    route() {
+      // return this.$route.path.split('/').filter((v: String) => v !== '')
```
[the remark this answers is not in the document]
Comment (baymax-codecare) [reply components/views/settings/breadcrumbs/Breadcrumbs.vue]: @stavares843 just removed that line and committed.

**Review by WanderingHogan — APPROVED** [review]
very nice!

**Comment by phillsatellite** [comment]
Only little question this might be a little picky, the breadcrumb for audio and video/accounts and devices both just say Settings -> Audio and Settings -> Accounts, should they be Settings -> Audio & Video / Settings -> Accounts & Devices? Otherwise looked perfect 
Screen Shot 2022-02-23 at 2 05 43 PM
Screen Shot 2022-02-23 at 2 06 03 PM

**Comment by baymax-codecare** [comment]
oh my gash, after rebase, why all the other's commits here? @stavares843 @phillsatellite

**Comment by baymax-codecare** [comment]
thought, my change & rebase make an error.

File: components/views/settings/modal/Modal.html [hunk 2]
```diff
@@ -23,13 +23,17 @@
...
           <SettingsPagesPersonalize
             v-if="this.ui.settingsRoute === 'personalize'"
           />
-          <SettingsPagesAccounts v-if="this.ui.settingsRoute === 'accounts'" />
-          <SettingsPagesAudio v-if="this.ui.settingsRoute === 'audio'" />
+          <SettingsPagesAccounts
+            v-if="this.ui.settingsRoute === 'accounts & Devices'"
+          />
+          <SettingsPagesAudio
+            v-if="this.ui.settingsRoute === 'audio & Video'"
```
Comment (molimauro) [inline components/views/settings/modal/Modal.html:35]: can we transform these routes string in constants and import them where needed? Im worried that strings like "accounts & Devices" can cause subtle bugs in the future. Thanks!

**Comment by stavares843** [comment]
added changes requested due to mauro's review

**Comment by molimauro** [comment]
Thanks @baymax-codecare, great work!

**Review by molimauro — APPROVED** [review]

**Comment by molimauro** [comment]
love this action

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**